### PR TITLE
Add helpers and demo

### PR DESCRIPTION
**Pull request by notwaldorf**
I've added the 3 helpers we've been working on, and a couple of demos to illustrate them.
Is this PR too enormous? Do you want me to break it up?

Still to do (in a followup PR):
- `lazyReducerEnhancer` depends on `combineReducers` and I want to separate that
- the demo is using `import` and I need to fix that or add a shim

### CODE REVIEW

File: lazyReducerEnhancer.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+*/
+
+// TODO: We are using this version because standard Redux uses names not paths.
+// Investigate if we can require the user to provide their version of combineReducers.
+import combineReducers from '../node_modules/@0xcda7a/redux-es6/es/combineReducers.js';
```
[the remark this answers is not in the document]
Comment (notwaldorf) [reply lazyReducerEnhancer.js]: oops i forgot to check it in! done

File: lazyReducerEnhancer.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  enables that.
+
+  Sample use (where you define your redux store, in store.js):
+
+  import lazyReducerEnhancer from '../node_modules/@webcomponents/redux-helpers/lazyReducerEnhancer.js';
+  import someReducer from './reducers/someReducer.js';
+
+  export const store = createStore(
+    (state, action) => state,
+    compose(lazyReducerEnhancer, applyMiddleware(thunk))
```
Comment (frankiefu) [inline lazyReducerEnhancer.js]: Update example usage, e.g. `compose(lazyReducerEnhancer(combineReducers), applyMiddleware(thunk))`

File: package.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+  "name": "redux-helpers",
+  "version": "",
+  "contributors": [
+    "The Polymer Authors"
+  ],
+  "license": "https://polymer.github.io/LICENSE.txt",
+  "dependencies": {
+    "@0xcda7a/redux-es6": "^3.7.2",
```
Comment (keanulee) [inline package.json]: devDependency?

File: router.js [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+  import { navigate } from '../actions/app.js';
+
+  class MyElement extends connect(store)(HTMLElement) {
+    // ...
+
+    connectedCallback() {
+      // If you don’t have any other work to do other than dispatching an action,
+      // you can write something like:
+      installRouter(() => store.dispatch(navigate(window.location)));
```
Comment (keanulee) [inline router.js]: `installRouter` is actually generic enough to be used outside of custom elements. For example, in polymer-redux-hn, this is just called outside of the element at the top-level of hn-app.js.

File: package.json (hunk 3 above)
Comment (notwaldorf) [reply package.json]: yup, now that i've moved `combineReducers` away, good call.

**Pull request merged**